org.eclipse.scanning.api/src/org/eclipse/scanning/api/points/models/AbstractMapModel.java

```diff
@@ -61,6 +64,16 @@ public void setSlowAxisName(String newValue) {
 		this.pcs.firePropertyChange("slowAxisName", oldValue, newValue);
 	}
 
+	@Override
+	public boolean isContinuous() {
+		return continuous;
+	}
+
+	@Override
+	public void setContinuous(boolean continuous) {
+		this.continuous = continuous;
+	}
+
 	@UiHidden
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "UiHidden"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)